challenge.js

```diff
@@ -57,5 +64,5 @@
  */
 
 // ANSWER
-
+// https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild
```

Comment:
Correct. Here is a snippet using `document.createElement` and this method to accomplish what you described in Q1:

```js
var div = document.querySelectorAll("body > div")[0]; // the zeroth element of the set of all divs that are immediate children of the body
var e = document.createElement('textarea'); // this textarea element is not on the DOM yet
div.appendChild(e); // mission accomplished
```

1